libs/utils/android/workloads/geekbench.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2015, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+import os
+import logging
+from time import sleep
+
+from subprocess import Popen, PIPE
+
+from android import Screen, Workload, System
+
+class Geekbench(Workload):
+    """
+    Android Geekbench workload
+    """
+
+    # Package required by this workload
+    package = 'com.primatelabs.geekbench'
+    activity = '.HomeActivity'
+
+    def __init__(self, test_env):
+        super(Geekbench, self).__init__(test_env)
+        self._log = logging.getLogger('Geekbench')
+        self._log.debug('Workload created')
+
+    def run(self, out_dir, test_name, collect=''):
+        """
+        Run single Geekbench workload.
+
+        :param out_dir: Path to experiment directory where to store results.
+        :type out_dir: str
```

Comment:
Can you clarify that this is the directory on the host. I guess it's obvious when you think about it but it's nice not to have to think about it :P